app/helpers/assignment_result_helper.rb

```diff
@@ -3,6 +3,10 @@ def t_expectation(expectation)
     raw Mumukit::Inspection::Expectation.parse(expectation).translate
   end
 
+  def humanized_expectation_result_item(expectation)
+    %Q{<li>#{status_icon(expectation[:result])} #{t_expectation expectation}</li>}.html_safe
```

Comment:
It's just a minor detail, but a content_tag :li may make this line more readable

Reply:
we tried using a content tag but it would end up escaping the content and showing it all as a string, and having it not escape it would require calling it as `content_tag :li, content, {}, false` which didn't look that good tbh